gems/aws-sdk-s3/spec/object/upload_file_spec.rb

```diff
@@ -94,6 +94,12 @@ module S3
             object.upload_file(ten_meg_file.path)
           end
 
+          it 'yields the response to the given block' do
```

Comment:
I don't think two tests make sense - this feature should work irregardless of large objects or small objects. You can move this test "up one level" out of the 'describe' block and in its own 'it' block, then delete the other test.